generic-ec/src/arithmetic.rs

```diff
@@ -10,6 +10,9 @@ mod laws {
     };
     use crate::{Point, Scalar};
 
+    pub trait AlwaysNonZero {}
+    impl<T> AlwaysNonZero for NonZero<T> {}
```

Comment:
Missing docs

Also, why not make this a trait with methods? Maybe add something like `invert` and `safe_divide`

Reply:
Ah-huh, I completely missed the "laws" context. Never seen this module

Reply:
This is internal trait, it's not exposed